<?xml version="1.1" encoding="UTF-8"?>
<launch>
    <node pkg="dynamic_tracker" name="cluster_tester" type="cluster_test_node" output="screen">
        <!-- dynamic tracking -->
        <param name="tracking_update_timeout" value="1.0" type="double"/>
        <param name="tracking_update_rate" value="10.0" type="double"/>
        <param name="tracking_ts" value="0.35" type="double"/>
        <param name="pose_type" value="2" type="int"/>
        <param name="cluster_type" value="2" type="int"/>
        <param name="clip_height" value="0.1" type="double"/>
        <param name="cloud_topic" type="string" value="/cloud_registered"/>
        <param name="pose_topic" type="string" value="pose"/>
        <param name="odom_topic" type="string" value="/iris/mavros/local_position/odom"/>

        <!-- cluster -->
        <param name="cluster_pts" value="4" type="int" />
        <param name="epsilon" value="1.5" type="double"/>
        <param name="adaptive" value="true" type="bool" />


    </node>

    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find dynamic_tracker)/config/default.rviz" />

    <node pkg="rosbag" type="play" name="player"  args="--clock /home/huyu/bags/gazebo2long.bag"/>

</launch> 